### PR TITLE
Disable Trash feature and Verify

### PR DESCRIPTION
**Pull request by preethisn**
Description -Disable Trash feature, Update the config settings and  delete images and check if they are not moved to trash automatically

Added this test coverage in RBD regression suits

### CODE REVIEW

**Comment by preethisn** [comment]
Logs are uploaded in -> http://magna002.ceph.redhat.com/ceph-qe-logs/Preethi/cephci-run-S0VCVQ/

File: tests/rbd/rbd_trash.py [hunk 1]
```diff
@@ -30,9 +30,11 @@ def run(**kw):
     2. generate IO in images
     3. Enable "rbd_move_to_trash_on_remove" in client config
     4. Delete images and verify the deleted images are moved to trash automatically.
+    5. Disable "rbd_move_to_trash_on_remove" in client config and verify images are not moved to trash automatically
```
Comment (VasishtaShastry) [inline tests/rbd/rbd_trash.py:33]: Can we please update module doc with 
1) NEw Testcae id and summary
2) clarify in flow that 4th and 5th step are for different TCs ?

File: tests/rbd/rbd_trash.py [hunk 2]
```diff
@@ -43,14 +52,24 @@ def run(**kw):
...
+                raise ImageNotFoundError(" Deleted Image not found in the Trash")
+        else:
+            rbd.create_image(pool_name=pool, image_name=image, size=size)
+            client = kw["ceph_cluster"].get_nodes(role="client")[0]
+            run_fio(image_name=image, pool_name=pool, client_node=client)
+            cmd = "ceph config set client rbd_move_to_trash_on_remove false"
+            rbd.exec_cmd(cmd=cmd)
+            rbd.remove_image(pool, image)
+            if rbd.trash_exist(pool, image):
+                raise ImageFoundError(" Image is found in the Trash")
```
Comment (VasishtaShastry) [inline tests/rbd/rbd_trash.py:72]: many lines here are replicated, can we try to optimise?

**Review by VasishtaShastry — CHANGES_REQUESTED** [review]

**Comment by preethisn** [comment]
Addressed the review comments, latest logs are attached
http://magna002.ceph.redhat.com/ceph-qe-logs/Preethi/cephci-run-UQZNJ7/

**Review by VasishtaShastry — APPROVED** [review]

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.

**Pull request merged**